src/Raven.Server/RavenServer.cs

```diff
@@ -494,16 +493,35 @@ public bool RefreshClusterCertificate(object state)
                     var certUpdate = ServerStore.Cluster.GetItem(context, "server/cert");
                     if (certUpdate != null)
                     {
-                        // we are already in the process of updating the certificate, so we need
-                        // to nudge all the nodes in the cluster to check the replacement state.
-                        // If a node confirmed but failed with the actual replacement (e.g. file permissions)
-                        // this will make sure it will try again in the next round (1 hour).
-                        await ServerStore.SendToLeaderAsync(new RecheckStatusOfServerCertificateCommand());
-                        return;
+                        if (certUpdate.TryGet("Confirmations", out int confirmations) == false)
+                            throw new InvalidOperationException("Expected to get confirmations count");
+
+                        if (certUpdate.TryGet("Replaced", out int replaced) == false)
```

Comment:
Isn't this a new property? If so, what will happen in a cross versioned cluster?

Reply:
Yes that's a problem, will fix.